research/picai/fl_nnunet/nnunet_client.py

```diff
@@ -362,7 +379,7 @@ def setup_client(self, config: Config) -> None:
 
     def predict(self, input: TorchInputType) -> Tuple[TorchPredType, Dict[str, torch.Tensor]]:
         """
-        Generate model outputs. Overridden because nnunets output lists when
+        Generate model outputs. Overridden because nnunet's output lists when
```

Comment:
I actually think the previous version of this was correct here 🙂. I don't care too much either way though ha